### PR TITLE
Graphql Blog - added blog content + assests + author desc

### PR DESCRIPTION
**Pull request by anup-a**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] You created a branch from `dev` branch
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] Checked for all grammatical errors in the blog.
- [x] Checked for any duplicate blog entry.
- [x] You PR is raised on `dev` branch and not on`master`.

##### Description of change

Posting a new blog "Rise and Rise of GraphQL", based on this issue https://github.com/LoginRadius/engineering-portal/issues/233

##### Author Details for Blog Post Entry

id: Anup Aglawe
bio: A computer science and engineering undergraduate student at IIT Gandhinagar. I immensly interested in developing applications and utlities.
github: "anup-a"
linkedin: "anup-aglawe-294087148"

### CODE REVIEW

File: content/blog/rise-and-rise-of-graphql/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    age
+  }
+}
+```
+
+## Cons
+
+Despite its powers, you might not need graphQL in your applications -
+
+- Since it adds unnecessary overhead and complex architecture for simple Applications
```
Comment (vyasriday) [inline content/blog/rise-and-rise-of-graphql/index.md:89]: please change Applications to applications. Rest everything looks good

**Review by vyasriday — CHANGES_REQUESTED** [review]

File: content/blog/rise-and-rise-of-graphql/index.md (hunk 1 above)
Comment (anup-a) [reply content/blog/rise-and-rise-of-graphql/index.md]: Done   👍

Comment (vyasriday) [reply content/blog/rise-and-rise-of-graphql/index.md]: Thanks

**Comment by vyasriday** [comment]
Merged PR. Thanks for contributing @anup-a .

**Comment by vyasriday** [comment]
I hope you have gone through https://www.loginradius.com/engineering/page/hacktoberfest2020 and filled your details for LoginRadius swags.

**Pull request merged**